uint8 id

string robot_name1
string robot_name2
uint8 id1
uint8 id2